perf: better trkr algorithm

wise-introvert opened the pull request:
Made `generateTRKR()` algo more linear, making it more performant :wink: 

BTW: an awesome idea, executed perfectly. Kudos to you!

wise-introvert committed:
perf: better trkr algorithm

Make `generateTRKR()` algo more linear, making it more performant ;)

rayarindam2111 commented on public/umang-cowin.js:
```diff
@@ -53,9 +53,21 @@ class umangWorker {
+                        .slice(0, 2)
+                        .join("")
+                        .toUpperCase()
+                        .replace(/^(.)|\s+(.)/g, (c) =>
```
`.replace(/^(.)/g, (c) =>`

You can just select the first letter and not go for the whitespace + 1st letter combo, since that case never arises here.

wise-introvert replied:
@rayarindam2111 I've updated the [algo](https://github.com/rayarindam2111/Co-WIN-automated-slot-booking/pull/6/commits/250a53c1fc2b7787b91a8d4801691dbca5cb7bac)

rayarindam2111 reviewed (changes requested):
Thank you for your appreciation of the project. If it helped you, consider leaving a star and sending it to other people who might need it :)

wise-introvert committed:
refactor: update regex

Based on https://github.com/rayarindam2111/Co-WIN-automated-slot-booking/pull/6#discussion_r643881210
